src/components/FileTree/index.tsx

```diff
@@ -184,4 +193,26 @@ export class FileTreeBase extends React.Component<Props> {
   }
 }
 
-export default connect()(FileTreeBase) as React.ComponentType<PublicProps>;
+const mapStateToProps = (
+  state: ApplicationState,
+  ownProps: PublicProps,
+): PropsFromState => {
+  const { versionId } = ownProps;
+  const version = getVersionInfo(state.versions, versionId);
+
+  if (!version) {
+    // This should never happen as we are fetching the version in all of the
+    // parents on this component and only rendering the FileTree if we have a
+    // version, but let's log a warning in case we encounter a case where this
+    // does happen.
+    log.warn(`No version was loaded for version: `, versionId);
```

Comment:
You asked if we should add a test for this. Since we use TypeScript, I'm not worried about it breaking so a test is not very helpful. However, it decreases our code coverage score which is a bit annoying. I guess you could add a test just to keep code coverage up? Meh.